src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days >= 3 && days <= 6) {
```

Comment:
```suggestion
  if (days <= 6) {
```
То что days не >= 3 проверяет предыдущий if, и если он меньше трех - до этого if дело не дойдет